miqa/core/conversion/json_to_csv.py

```diff
@@ -23,7 +23,7 @@ def jsonObjectToCsvContent(jsonObject):
         expId = scan['experiment_id']
 
         scanPath = scan['path']
-        pathComps = scanPath.split(os.path.sep)
+        pathComps = scanPath.rstrip(os.path.sep).split(os.path.sep)
```

Comment:
Would all this path manipulation logic be easier if we just made `scanPath` a `pathlib.Path`? Then we could just use its `parts` property.